feat: allow building stable base snaps (#6177)
snapcore/snapcraft: docs/release-notes/snapcraft-8-15.rst

.. _release-8.15:

Snapcraft 8.15 release notes
============================

14 April 2026

Learn about the new features, changes, and fixes introduced in Snapcraft 8.15.

Requirements and compatibility
------------------------------
See :ref:`reference-system-requirements` for information on the minimum hardware and
installed software.

What's new
----------

Snapcraft 8.15 brings the following features, integrations, and improvements.

> Support for bootstrapping stable base snaps
> ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Reviewer: ```suggestion
Bootstrapping stable base snaps
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

> 

Reviewer: It might be wise to connect the dots with a "Previously, base snaps..." paragraph at the start of this note.

> To allow bootstrapping new bases, base snaps can now be built with ``grade: stable``,

Reviewer: ```suggestion
Base snaps such as core26 can now bootstrap themselves with ``grade: stable``,
```

even when using the ``devel`` build base.

Minor features
--------------

Snapcraft 8.15 brings the following minor changes.

> Improved support for packages in the Colcon plugin
> ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Reviewer: What do you think about:

```suggestion
More flexible stage packages with the Colcon plugin
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

I can't think of a way to shorten it. There's no hard cap, but it's ideal when the heading doesn't wrap in the page nav.

> The :ref:`reference-colcon-plugin` now allows you to stage packages from ROS
> content sharing snaps that have no upstream release.

Reviewer: ```suggestion
The :ref:`reference-colcon-plugin` can now stage packages from ROS
content sharing snaps that have no upstream release.
```
